recipes/blessings/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "blessings" %}
+{% set version = "1.6" %}
+{% set sha256 = "edc5713061f10966048bf6b40d9a514b381e0ba849c64e034c4ef6c1847d3007" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
```

Comment:
Why skip Windows? I know one needs something like `colorama` as well, but shouldn't this still work on Windows?